v03_pipeline/lib/reference_data/gencode/download_utils.py

```diff
@@ -0,0 +1,112 @@
+import logging
```

Comment:
All the `gencode` files are also copy-pasta to preserve the deployability of v2.